### DISCUSSION TITLE
Does serverless support node 18?

### DISCUSSION THREAD

**Discussion by bitcompass** [serverless/serverless#11583] [General]
I am having issues to make my serverless project runs on node 18.
It works with Node 12, Node 14 and Node 16 but not 18.
Is it possible to run serverless project on node 18?
Please advise.

**Comment by pgrzesik** [comment]
Hello @devwiz1028 👋 Could you share a bit more what are the issues that you're running into? One of the main changes is switch from bundled AWS SDK JS v2 to v3 in Node 18, maybe that's causing the problem for you?
